test/validate/validate.go

```diff
@@ -154,77 +167,121 @@ func (v *Validator) validateIPs(ctx context.Context, stateFileIps stateFileIpsFu
 		if err != nil {
 			return errors.Wrapf(err, "failed to exec into privileged pod - %s", podName)
 		}
-		filePodIps, err := stateFileIps(result)
+		cnsPodIPs, err := stateFileIps(result)
```

Comment:
`validateIPs()` is used for multiple state files, so a general var name `filePodIps` was used.